snuba/admin/static/table.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ReactNode } from "react";
+
+import { COLORS } from "./theme";
+
+type TableProps = {
+  headerData: ReactNode[];
+  rowData: ReactNode[][];
```

Comment:
Does this mean that a table element can also be a component?

Reply:
So I can but buttons and other wacky stuff in there?

Reply:
Yeah, the table element is a component. The idea was that we can just pass anything with any kind of formatting and style into the rowData array and it will stick it all into the table cells.